components/chat/chat-hooks/use-chat-handler.tsx

```
@@ -427,6 +430,23 @@ export const useChatHandler = () => {
         ? { newChat: !currentChat }
         : { id: chatId, newChat: !currentChat };
 
+      // Create chat early if it doesn't exist
+      if (!currentChat && !isTemporaryChat) {
+        currentChat = await handleCreateChat(
+          baseModel!,
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)